compose/backend.yml

```diff
@@ -165,23 +162,25 @@ services:
       - ccd-docker-ccd-shared-database-data:/var/lib/postgresql/data
 
   shared-db:
-    image: hmcts.azurecr.io/hmcts/sidam/sidam-idm-pg:1.0.0
+    image: "hmctsprivate.azurecr.io/idam/shared-db:latest"
     ports:
       - 5051:5432
+    volumes:
+      - ccd-docker-idam-shared-database-data:/var/lib/postgresql/data  
 
-  am-database:
-    image: hmcts.azurecr.io/am-db
-    container_name: access_management_db
-    environment:
-      AM_DB:
-      AM_DB_USERNAME:
-      AM_DB_PASSWORD:
-    healthcheck:
-      interval: 10s
-      timeout: 10s
-      retries: 10
-    ports:
-      - 5600:5432
+  # am-database:
```

Comment:
please add a comment explaining that this commenting out is just temporary and the reason